jbmc/src/java_bytecode/ci_lazy_methods.cpp

```diff
@@ -297,6 +271,40 @@ bool ci_lazy_methodst::operator()(
   return false;
 }
 
+std::unordered_set<irep_idt>
+ci_lazy_methodst::reachable_from_main_function(const symbol_tablet &symbol_table)
```

Comment:
You might want to document here the fact that is there is no main method it gets all the methods from the main class (and if no main class, all the menthods!)